make/helm.mk

```diff
@@ -1,5 +1,10 @@
 ##@ Helm Charts
 
+# Chart name
+CHART_NAME ?= authorino-operator
```

Comment:
It looks like nothing here refers to `$(REPO)` anymore, so OK with this change. Let's just keep an eye in case there was something elsewhere relying on the old variable name, even though it's very unlikely being the name of the project.